sonar-duplications/src/main/java/org/sonar/duplications/internal/pmd/TokenizerBridge.java

```diff
@@ -54,8 +52,10 @@ public TokenizerBridge(Tokenizer tokenizer, int blockSize) {
     TokenEntry.clearImages();
     try {
       tokenizer.tokenize(sourceCode, tokens);
-    } catch (IOException e) {
-      throw Throwables.propagate(e);
+    } catch (RuntimeException e) {
+      throw e;
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)